crates/uv-resolver/src/resolver/indexes.rs

```diff
@@ -20,7 +20,13 @@ use crate::{DependencyMode, Manifest, ResolverEnvironment};
 ///
 /// [`Indexes`] would contain a single entry mapping `torch` to `https://download.pytorch.org/whl/cu121`.
 #[derive(Debug, Default, Clone)]
-pub(crate) struct Indexes(ForkMap<IndexUrl>);
+pub(crate) struct Indexes(ForkMap<Entry>);
+
+#[derive(Debug, Clone)]
+struct Entry {
+    index: IndexUrl,
+    conflict: Option<ConflictItem>,
```

Comment:
This is the subset of forks in which this entry occurs, not a conflict we generated?